test/linter/test-versions.js

```diff
@@ -50,7 +50,7 @@ for (const browser of Object.keys(browsers)) {
 }
 
 /** @type {string[]} */
-const blockMany = [
+const realValuesTargetBrowsers = [
```

Comment:
Is this essentially `browsersRequiringNumericalVersions`? Or what does the "real" refer to?

Reply:
Related question: Why do onlyu some browsers require "real" values?

Reply:
That's correct -- we've used the terms "real" and "nonreal" to describe exact version numbers and boolean/null values respectively throughout BCD!